.github/workflows/release.yml

```diff
@@ -27,6 +27,5 @@ jobs:
           github-token: ${{ secrets.github_token }}
           npm-token: ${{ secrets[format('NPM_TOKEN_{0}', github.actor)] || secrets.NPM_TOKEN }}
           optic-token: ${{ secrets[format('OPTIC_TOKEN_{0}', github.actor)] || secrets.OPTIC_TOKEN }}
-          commit-message: ${{ github.event.inputs.commit-message }}
```

Comment:
The `commit-message` input above is no more needed in this case

Reply:
let's also remove tag input and npm-tag parameter

Reply:
Done